packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5642,10 +5647,7 @@ export default class Meeting extends StatelessWebexPlugin {
             this.unsetPeerConnections();
           }
 
-          LoggerProxy.logger.error(
-            `${LOG_HEADER} Error adding media failed to initiate PC and send request, `,
-            error
-          );
+          LoggerProxy.logger.error(`${LOG_HEADER} failed to establish media connection: `, error);
```

Comment:
We should probably move this logger up above before the metrics call, or right after, but at the very least, before the stop analyzer and stop media connection stuff

Reply:
log moved to the top of the catch